doc/user/content/ingest-data/postgres/self-hosted.md

```diff
@@ -479,12 +569,66 @@ start by selecting the relevant option.
 
 ### 3. Monitor the ingestion status
 
+{{< tabs >}}
+
+{{< tab "Console">}}
+
+From the [**Database object explorer**](/console/data/) in the [Materialize
+console], you can see the status of the source you created.
+
+Before it starts consuming the replication stream, Materialize takes a snapshot of the relevant tables in your publication. Until this snapshot is complete, Materialize won't have the same view of your data as your PostgreSQL database.
```

Comment:
"Materialize won't have the same view of your data as your PostgreSQL database" suggests that the source can be queried while it's snapshotting — it can't. So we should adapt this sentence (in all guides) to instead make it clear that snapshotting must complete before you can query the source.